packages/react/src/components/Button/Button.js

```diff
@@ -20,6 +21,7 @@ const Button = React.forwardRef(function Button(
     className,
     disabled,
     small,
+    size,
```

Comment:
Should we add a prop type for this? Maybe:

```
size: PropTypes.oneOf(['field', 'small'])
```

Reply:
Also, instead of `small` should we refer to it like `short` or something that correlates with when this might be used? (like with field)

Small could end up be confusing potentially since it's a relative unit.

Reply:
@joshblack we're deciding to hold off on this property name change in this PR per https://github.com/carbon-design-system/carbon/pull/2719#issuecomment-496651187 but happy to continue discussion later!